junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/Constants.java

```diff
@@ -187,6 +205,205 @@
 	public static final String PARALLEL_CONFIG_CUSTOM_CLASS_PROPERTY_NAME = PARALLEL_CONFIG_PREFIX
 			+ CONFIG_CUSTOM_CLASS_PROPERTY_NAME;
 
+	/**
+	 * Property name used to set the default timeout for all testable and
```

Comment:
indentation for method-level Javadoc is all wonky in this class now.

mixture of spaces and tabs? 🤔

Reply:
Or... is that a new bug in the GitHub UI?

Reply:
Doesn't look wonky to me. Could you please post a screenshot?

Reply:
See above: likely just a bug in the display in the browser due to a change in GitHub (or my browser).

I checked the _source_, and it's fine.

Sorry for the false alarm.